src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -168,109 +194,112 @@ export const TradeConfirm = ({ history }: RouterProps) => {
                 />
               </Card.Heading>
             </WithBackButton>
-            <AssetToAsset
-              buyIcon={trade.buyAsset.icon}
-              tradeFiatAmount={tradeFiatAmount}
-              trade={trade}
-              mt={6}
-              status={sellTxid || isSubmitting ? status : undefined}
-            />
           </Card.Header>
           <Divider />
           <Card.Body pb={0} px={0}>
-            <Stack spacing={4}>
-              {sellTxid && (
+            <Stack
+              spacing={4}
+              borderColor={borderColor}
+              divider={<StackDivider />}
+              fontSize='sm'
+              fontWeight='medium'
+            >
+              <AssetToAsset
+                buyIcon={trade.buyAsset.icon}
+                sellIcon={trade.sellAsset.icon}
+                status={sellTxid || isSubmitting ? status : undefined}
+              />
+              <Stack spacing={4}>
                 <Row>
-                  <Row.Label>
-                    <RawText>Tx ID</RawText>
-                  </Row.Label>
+                  <Row.Label>Send</Row.Label>
```

Comment:
translation

Reply:
Addressed in https://github.com/shapeshift/web/pull/2584/commits/b1d5491d937fc68569f7781d2b1e93d27c47575c